### PR TITLE
DOCS-1720-IA-Registry-New

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
CC 18.0

Issue:
https://tigera.atlassian.net/browse/DOCS-1720

Link to docs preview:
https://deploy-preview-980--calico-cloud-preview-next.netlify.app/calico-cloud/next/image-assurance/scanners/registry-scanner/#get-registry-scanner-updates

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+description: Scan images in container registeries.
+---
+
+# Scan images in container registries
+
+Scan images in container registries at any time, on any infrastructure, including Kubernetes.
+
+## Value
+
+Add a layer of defense for images that don’t go through a pipeline (for example, third-party images), but are published to registries. If CVEs are missed in your build pipeline, you can catch them before they are deployed.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Minor: It sounds a bit more consistent at the end to say "published to a registry" instead of "published to registries" because it matches up with the singular "a pipeline".

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+## Concepts
+
+You can deploy the registry scanner wherever you want. Locally on a laptop. Anywhere. And it doesn’t have to be for Kubernetes.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: I think it reads better without the "Locally on a laptop. Anywhere." part. 

If anything, I think what might be missing in this section is a call out that the only restriction on where you run the registry scanner is that it must be run somewhere where there is a container runtime. 

This highlights the fact that the registry scanner is **not** a simple binary like the CLI. In fact, I think we should include this point in the Required section of this document.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+To use this option, all you need to do is:
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: I would prefer instead of "option" we used a different word. Because when I read "option" I immediately think of configuration options. 

Whereas, IIUC the word "option" here means "version of our scanner".

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+- Specify the registry paths to the images you want to scan
+- Provide permissions for the scanner to access your registries
+- Get a token for access to the Image Assurance API
+
+Based on the paths you specify, the scanner recursively scans all images in the registry once, and sends results to the Image Assurance dashboard in Manager UI.
+
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx:23]: We might also want to explicitly mention somewhere in this document that if the user wants to deploy the registry scanner as a pod in their Kubernetes cluster, they will need to create a manifest (perhaps we could even recommend using a Kubernetes [cronjob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/)) themselves. 

That exercise is left for the user and is outside the scope of this document.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+## Before you begin
+
+**Required**
+
+- Valid registry credentials to set up handshake with the registry scanner
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Why is the word "handshake" here? I think it has a specific meaning in terms of protocol-level terminology (e.g. SSL handshake). I am not sure it adds anything meaningful here and if anything gives me pause, as a reader, to ponder what exactly it's trying to get at.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+**Supported registry platforms**
+
+- Amazon ECR
+- Azure Container Registry
+- Docker, using a container runtime
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: What about the fact that the registry scanner supports GCR? 

It wasn't part of the requirements, but should we include here if we support it?

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Set up registry scanner](#set-up-registry-credentials)
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
+- [Troubleshoot](#troubleshoot)
+- [Get registry scanner updates](#get-registry-scanner-updates)
+
+### Download the registry scanner
+
+The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:v1.10.4`.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: This needs to use the `imageassuranceversion` variable so that the version is up-to-date from release to release.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+### Set up registry scanner
+
+The registry scanner requires authentication to access a registry. You can use Docker credentials, or Azure/AWS credentials.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: I am detecting a bit of confusion here from reading the instructions. 

When I read this, it sounds like I can use either Docker credentials OR Azure/AWS credentials. But that is not true is it? 

I cannot use **only** Azure/AWS credentials without the Docker credentials, can I?

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+
+#### Docker credentials
+
+Scanning a registry requires valid Docker credentials using these environment variables: `DOCKER_USERNAME` and `DOCKER_PASSWORD`.
+The registry scanner scan images only with tags; untagged images residing in your image registry are not pulled and scanned.
+
+If you have a valid /.docker/config.json, you can also mount this config file on the container while running the registry scanner.
+
+```
+docker run -e ... -v ~/.docker/config.json:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: This needs to use the imageassuranceversion variable so that the version is up-to-date from release to release.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+ -e IMAGE_ASSURANCE_API_TOKEN=$TOKEN -e DOCKER_USERNAME=<username> -e DOCKER_PASSWORD=<password>
+ quay.io/tigera/image-assurance-registry-scanner:v1.10.4
+```
+
+Example run for an acr registry with Azure credentials:
+
+```
+docker run -e REGISTRY=your-org.azurecr.io -e IMAGE_ASSURANCE_API_URL=https://<clustId>-management.dev.calicocloud.io/bast
+ -e IMAGE_ASSURANCE_API_TOKEN=$TOKEN -e AZURE_CLIENT_ID=<azure-client-id> -e AZURE_CLIENT_SECRET=<azure-client-secret> -e AZURE_TENANT_ID=<azure-tenant-id>
+ quay.io/tigera/image-assurance-registry-scanner:v1.10.4
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: This needs to use the {{imageassuranceversion}} variable so that the version is up-to-date from release to release.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+```
+
+Example run for an ecr registry with AWS credentials:
+
+```
+docker run -e REGISTRY=gcr.io -e IMAGE_ASSURANCE_API_URL=https://<clustId>-management.dev.calicocloud.io/bast
+ -e IMAGE_ASSURANCE_API_TOKEN=$TOKEN -e AWS_ACCESS_KEY_ID=<aws-access-key> -e AWS_SECRET_ACCESS_KEY=<aws-secret-access-key> -e AWS_REGION=<aws-region>
+ quay.io/tigera/image-assurance-registry-scanner:v1.10.4
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: This needs to use the {{imageassuranceversion}} variable so that the version is up-to-date from release to release.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+```
+
+Example run when mounting the dockerfile:
+
+```
+docker run -e REGISTRY=gcr.io -e IMAGE_ASSURANCE_API_URL=https://<clustId>-management.dev.calicocloud.io/bast
+ -e IMAGE_ASSURANCE_API_TOKEN=$TOKEN -v ~/.docker/config.json/:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: This needs to use the {{imageassuranceversion}} variable so that the version is up-to-date from release to release.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 13]
```diff
@@ -0,0 +1,150 @@
...
+
+### Troubleshoot
+
+- For issues authenticating with the registry, verify that the registry credentials are correct.
+- For issues uploading the results to the {{prodname}}, check that the API token and URL are correct.
+- Because the scanner is not running continuously, it may take some time for results to display in the Image Assurance dashboard, especially if you have many images or large images.
+
+### Get registry scanner updates
+
+For the latest registry scanner versions, see [quay repository](https://quay.io/organization/tigera).
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: I think we can keep this document up-to-date with the latest version of registry scanner using the {{imageassuranceversion}} variable from release to release.

So we can reword this part to be something like "To see previous versions of registry scanner ...".

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 14]
```diff
@@ -0,0 +1,150 @@
...
+To filter out images in the dev "sub" registry, use:  `-e REGISTRY_FILTER=prod-env,staging-env`.
+
+### Scan images and send results to {{prodname}}
+
+Example run for a gcr registry with Docker Username and Password:
+
+```
+docker run -e REGISTRY=gcr.io -e IMAGE_ASSURANCE_API_URL=https://<clustId>-management.dev.calicocloud.io/bast
+ -e IMAGE_ASSURANCE_API_TOKEN=$TOKEN -e DOCKER_USERNAME=<username> -e DOCKER_PASSWORD=<password>
+ quay.io/tigera/image-assurance-registry-scanner:v1.10.4
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: This needs to use the {{imageassuranceversion}} variable so that the version is up-to-date from release to release.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 15]
```diff
@@ -0,0 +1,150 @@
...
+Example run when mounting the dockerfile:
+
+```
+docker run -e REGISTRY=gcr.io -e IMAGE_ASSURANCE_API_URL=https://<clustId>-management.dev.calicocloud.io/bast
+ -e IMAGE_ASSURANCE_API_TOKEN=$TOKEN -v ~/.docker/config.json/:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
+```
+
+### Troubleshoot
+
+- For issues authenticating with the registry, verify that the registry credentials are correct.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: We might also want to call out that if the images scanned by the registry scanner do not match what is expected, then confirm that the corresponding credentials have the right permission level on the registry side.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 16]
```diff
@@ -0,0 +1,150 @@
...
+- For issues uploading the results to the {{prodname}}, check that the API token and URL are correct.
+- Because the scanner is not running continuously, it may take some time for results to display in the Image Assurance dashboard, especially if you have many images or large images.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Can you clarify here what "scanner" you're referring to? This doesn't sound like you're describing the registry scanner. 

If you are, then I find the "not running continuously" curious. Technically once it starts executing, it is running continuously, until it was scanned all of the images. 

So from a user perspective, I'm not quite sure what this is trying to say.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 17]
```diff
@@ -0,0 +1,150 @@
...
+The registry scanner currently scans one registry at a time. If you specify the Azure or AWS credentials, the Docker username and password will not be used.
+
+#### Required inputs
+
+The registry scanner requires:
+
+- `REGISTRY` - the registry you want to scan. (gcr.io)
+- `IMAGE_ASSURANCE_API_URL` - Get the URL in the Manager UI.
+- `IMAGE_ASSURANCE_API_TOKEN` - Get the token in the Manager UI.
+- Either the `DOCKER_USERNAME` and `DOCKER_PASSWORD`, Azure credentials, or AWS credentials.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: For clarity, it would be nice to see an "or" between "`DOCKER_USERNAME` and `DOCKER_PASSWORD`" and  "Azure credentials".

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 18]
```diff
@@ -0,0 +1,150 @@
...
+The registry scanner scan images only with tags; untagged images residing in your image registry are not pulled and scanned.
+
+If you have a valid /.docker/config.json, you can also mount this config file on the container while running the registry scanner.
+
+```
+docker run -e ... -v ~/.docker/config.json:/.docker/config.json quay.io/tigera/image-assurance-registry-scanner:v1.10.4
+```
+#### Azure and AWS credentials
+
+The registry scanner also supports authentication using AZURE and AWS credentials. The registry scanner will retrieve all images without tags.
```
Comment (stevegaossou) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: I think it would be good to have a more explicit call out here for why the user might want to use Azure / AWS credentials. 

IIUC a user **must** use Docker credentials and then on top of that, if the user wants to scan all untagged images, then a user **must** use corresponding Azure or AWS credentials as well. Is this correct? 

If so we really need to be more clear on this here.

**Review by stevegaossou — COMMENTED** [review]
Added comments.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 6 above)
Comment (WilliamTigera) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx:34]: Yes, actually change this to GCR instead of `docker, using a container runtime`.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 8 above)
Comment (WilliamTigera) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: I think you can leave out the last sentence because the remaining section below explains more in detail. @bmckercher123

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 16 above)
Comment (WilliamTigera) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: This was in reference to the scanner worker... I'm wondering how we should word this. 

Perhaps: `Images scanned using the registry scanner may take up to 5 minutes to appear on the Manager UI`?

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 18 above)
Comment (WilliamTigera) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: @bmckercher123 Please reword this to:

```suggestion
The registry scanner also supports authentication using AZURE and AWS credentials. If users use these credentials instead of the Docker Username and Password, they will be able to retrieve untagged images for these registries.
```

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 6 above)
Comment (WilliamTigera) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]:
```suggestion
- GCR
```
Please remove Docker here.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 5 above)
Comment (WilliamTigera) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]:
```suggestion
- Valid registry credentials
```

I believe we don't need to add the handshake part as Steve says.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Changed to: You can run the registry scanner wherever there is a container runtime.

Added this:

**Required**

- Registry scanner is running where there is a container runtime
- Valid registry credentials

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed. Changed to: registry scanner

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 4 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx:23]: Added to Concepts section.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 5 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 6 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Updated.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 7 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Updated all occurrences

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 9 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 10 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 11 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 12 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 13 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 14 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 15 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Added.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 19]
```diff
@@ -0,0 +1,167 @@
...
+
+### Download the registry scanner
+
+The registry scanner comes in a Docker image. To get the image, run this command: `docker pull quay.io/tigera/image-assurance-registry-scanner:{{imageassuranceversion}}`.
+
+### Set up authentication to registry scanner
+
+The registry scanner requires authentication to access a registry. Set up credentials for one of the following registry platforms: Docker/Google, Azure, or AWS.
+
+#### Docker/GRC required credentials
```
Comment (WilliamTigera) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx:65]:
```suggestion
#### Docker/GCR required credentials
```

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 20]
```diff
@@ -0,0 +1,167 @@
...
+
+All images (tagged and untagged) are scanned. Registry instances are scanned one at a time. If Docker credentials are found, they are ignored.
+
+- `AZURE_CLIENT_ID`
+- `AZURE_CLIENT_SECRET` or `AZURE_FEDERATED_TOKEN`
+- `AZURE_TENANT_ID`
+
+#### AWS required credentials
+
+All images (tagged and untagged) are scanned. Registry istances are scanned one at a time. If Docker credentials are found, they are ignored.
```
Comment (WilliamTigera) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]:
```suggestion
All images (tagged and untagged) are scanned. Registry instances are scanned one at a time. If Docker credentials are found, they are ignored.
```

File: calico-cloud/variables.js [hunk 21]
```diff
@@ -16,7 +16,7 @@ const variables = {
   clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.16.1-5',
   clouddownloadbase: 'https://installer.calicocloud.io',
   cloudoperatorimage: 'quay.io/tigera/cc-operator',
-  imageassuranceversion: 'v1.5.6',
+  imageassuranceversion: 'v1.9.1',
```
Comment (WilliamTigera) [inline calico-cloud/variables.js]:
```suggestion
  imageassuranceversion: 'v1.10.9',
```

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 16 above)
Comment (stevegaossou) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: It is indeed a bit tricky to word this with enough information without getting into too much detail about what scanner worker is and how it relates to the registry scanner. 

Perhaps we can just say something like "Image scan results uploaded to Calico Cloud through the registry scanner   require additional processing before appearing in the UI. This may result in a time delay before CVE results appear for those images in the UI."

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 4 above)
Comment (stevegaossou) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx:23]: > ... we recommend that you create a manifest or use a Kubernetes cronjob.

Hmm, saying "create a manifest or cronjob" is confusing because a manifest just means the user needs to define a Kubernetes resource that will house the registry scanner. 

I think probably the type of resource that makes the most sense for the registry scanner is either a job or cronjob. Job would run the registry scanner once. Cronjob would allow the user to re-run the registry scanner on a regular interval. 

So I think our advice here is just to define a Kubernetes job or cronjob to deploy the registry scanner into a Kubernetes cluster.

Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx:23]: Fixed as noted.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 16 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 17 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 19 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx:65]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 20 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/variables.js (hunk 21 above)
Comment (bmckercher123) [reply calico-cloud/variables.js]: Updated version.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 18 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Updated at beginning of section for clarity.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 22]
```diff
@@ -0,0 +1,168 @@
...
+
+- The registry scanner is available as an image (not using Tigera operator installation)
+- You can scan images from only one of the supported registry platforms/account per instance
+- If you use the registry scanner with Docker, only tagged images are scanned. However, if you use the scanner with Amazon or Azure, all images (tagged and untagged) are scanned.
+
+## How to
+
+- [Download the registry scanner](#download-the-registry-scanner)
+- [Set up authentication to registry scanner](#set-up-authentication-to-registry-scanner)
+- [Set up registry scanner](#set-up-registry-credentials)
```
Comment (MosesTigera) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx:49]: - [Set up registry scanner](#set-up-registry-scanner)

Comment (MosesTigera) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: #set-up-registry-credentials => #set-up-registry-scanner

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx [hunk 23]
```diff
@@ -0,0 +1,168 @@
...
+- [Scan images and send results to Calico Cloud](#scan-images-and-send-results-calico-cloud)
```
Comment (MosesTigera) [inline calico-cloud/image-assurance/scanners/registry-scanner.mdx]: #scan-images-and-send-results-calico-cloud => #scan-images-and-send-results-to-calico-cloud

Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed.

File: calico-cloud/image-assurance/scanners/registry-scanner.mdx (hunk 22 above)
Comment (bmckercher123) [reply calico-cloud/image-assurance/scanners/registry-scanner.mdx]: Fixed

**Review by stevegaossou — APPROVED** [review]
